doc/contributing/glossary.md

```diff
@@ -2,38 +2,42 @@
 
 Just a list of acronyms I've run across in the Ruby source code and their meanings.
 
+**Note on casing:** When a term appears in Ruby's codebase as both a variable/type name and a concept, this glossary uses
+the lowercase form to match its usage in code (e.g., `bop` for Basic Operator, `cfp` for Control Frame Pointer).
+Terms that appear only as macros or concepts use uppercase (e.g., `BIN`, `YARV`, `ICLASS`).
+
 | Term | Definition |
 | ---  | -----------|
-| `BIN` | Basic Instruction Name. Used as a macro to reference the YARV instruction. Converts pop into YARVINSN_pop. |
-| `bop` | Basic Operator. Relates to methods like `Integer` plus and minus which can be optimized as long as they haven't been redefined. |
-| `cc` | Call Cache.  An inline cache structure for the call site. Stored in the `cd` |
-| `cd` | Call Data. A data structure that points at the `ci` and the `cc`.  `iseq` objects points at the `cd`, and access call information and call caches via this structure |
-| `cfp`| Control Frame Pointer. Represents a Ruby stack frame.  Calling a method pushes a new frame (cfp), returning pops a frame. Points at  the `pc`, `sp`, `ep`, and the corresponding `iseq`|
-| `ci` | Call Information.  Refers to an `rb_callinfo` struct. Contains call information about the call site, including number of parameters to be passed, whether it they are keyword arguments or not, etc. Used in conjunction with the `cc` and `cd`. |
-| `cref` | Class reference. A structure pointing to the class reference where `klass_or_self`, visibility scope, and refinements are stored. It also stores a pointer to the next class in the hierarchy referenced by `rb_cref_struct * next`. The Class reference is lexically scoped. |
+| BIN | Basic Instruction Name. Used as a macro to reference the YARV instruction. Converts pop into YARVINSN_pop. |
```

Comment:
`BIN()` is a name of a macro. It seems fair to use backquotes on it in a markdown file. If RDoc doesn't parse/render it correctly, I think we should fix RDoc instead.

Reply:
Ok it took me a while but https://github.com/ruby/rdoc/pull/1386 should support it

Reply:
Thanks! Can we leave these backquotes in this PR then?

Reply:
Sure I'll close this now then.